.eslintrc.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
     },
   },
   rules: {
+    'react/react-in-jsx-scope': 'off',
```

Comment:
We don't want to apply this to every file right, just the files in website? We can use the overrides config to be more selective in what this rule applies to.

Reply:
👍